packages/ra-test/README.md

```diff
@@ -100,6 +100,37 @@ it('should send the user to another url', () => {
 });
 ```
 
+### Using the 'renderWithRedux' wrapper function
+
+Instead of wrapping the component under test with the `TestContext` by yourself you can get use of all of the above options and test your components almost natively looking like using just `@testing-library/react` thanks to the `renderWithRedux` wrapper function.
```

Comment:
```suggestion
Instead of wrapping the component under test with the `TestContext` by yourself you can use of all of the above options and test your components almost like using just `@testing-library/react` thanks to the `renderWithRedux` wrapper function.
```

Reply:
I didn't find a way to edit your suggestion so I commited the changes as a separate commit. I agree with the suggested edits and believe the 'of' after 'by yourself you can use' part is not needed.